src/moj/components/button-menu/README.md

```diff
@@ -30,7 +30,7 @@ The JavaScript
 
 ```js
 new MOJFrontend.ButtonGroup({
-  container: $('.moj-button-group'),
+  container: document.querySelector('.moj-button-group'),
   mq: '(min-width: 45em)',
   buttonText: 'Actions'
 });
```

Comment:
Pretty sure this can be removed as it's incorrect! `new MOJFrontend.ButtonGroup` doesn't actually exist! This should likely have been removed when we updated the button menu component - the nunjucks and the js example are wrong. At a glance the args look correct

Reply:
Yeah might be worth a separate PR - its not critical, I'm not sure how many people read the component READMEs anyway, some of them don't even have one!